### PR TITLE
#49: [Popover] rename misleading type props "absolute" and "relative" (closes #49)

### LINKED ISSUE #49: [Popover] rename misleading type props "absolute" and "relative"
remove `absolute` and `relative` in favor of `attachToBody={true/false}`

/cc @giogonzo @francescogior

### PR DESCRIPTION
**Pull request by FrancescoCioria**
Issue #49

### CODE REVIEW

File: src/popover/Popover.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ const Popover = React.createClass({
   },
 
   isAbsolute() {
-    return this.getPopoverProps().type === 'absolute';
+    return this.getPopoverProps().attachToBody === true;
```
Comment (ascariandrea) [inline src/popover/Popover.js]: You can simply return the value of `attachBody`

**Comment by ascariandrea** [comment]
:+1:

File: src/popover/Popover.js (hunk 1 above)
Comment (FrancescoCioria) [inline src/popover/Popover.js]: I know, but I prefer it to as a boolean (`attachToBody` can be `undefined`)

Comment (ascariandrea) [inline src/popover/Popover.js]: You can resolve by using `!!` operator, like `!!this.getPopoverProps().attachToBody`. It always returns a boolean ;)

**Pull request merged**